pkg/remote/google/provider.go

```diff
@@ -55,10 +53,8 @@ func (p *GCPTerraformProvider) Version() string {
 	return p.version
 }
 
-func (p *GCPTerraformProvider) GetConfig() config.GCPTerraformConfig {
+func (p *GCPTerraformProvider) SetConfig(scopes []string) config.GCPTerraformConfig {
```

Comment:
`GetConfig` was ok, we are not setting anything in this function

Reply:
Well there's also nothing to get as during automated tests we have to set the config (scope)